lib/plugins/deploy/deploy.js

```diff
@@ -28,6 +28,9 @@ class Deploy {
           'finalize',
         ],
         options: {
+          conceal: {
```

Comment:
Could you please add the shortcut version as well?

Reply:
I am happy to add a shortcut version, perhaps  `c`. However considering this option is rather niche, I though we could reserve the `c` shortcut for one that is more popular.
As it stands the `--force` option doesn't have a shortcut either.

@horike37 @RafalWilinski Final word is yours 🙇.

Reply:
Makes sense @timstott. I think we don't need shortcut.

Reply:
Makes sense. It would be OK dropping the shortcut :+1: